src/Components/ServiceScene/Breakdowns/SortByScene.tsx

```diff
@@ -21,16 +21,39 @@ export class SortCriteriaChanged extends BusEventBase {
 export class SortByScene extends SceneObjectBase<SortBySceneState> {
   public sortingOptions = [
     {
-      value: 'changepoint',
-      label: 'Most relevant',
-      description: 'Smart ordering of graphs based on the data',
+      label: '',
+      options: [
+        {
+          value: 'changepoint',
+          label: 'Most relevant',
+          description: 'Smart ordering of graphs based on the data',
+        },
+        {
+          value: ReducerID.stdDev,
+          label: 'Widest spread',
+          description: 'Prioritize graphs that have changed the most',
```

Comment:
Can we at least let users know that this is stdDev? i.e. 'Prioritize graphs that have changed the most (standard deviation)'

Reply:
Plus this description isn't accurate, standard deviation is a measure of variation from the mean, a single outlier with large magnitude of difference from the mean could give a particular graph the largest stdDev, but otherwise have all values with the same value, "changing the most" is subjective, and if I wanted to dig into why a particular graph is being sorted first with this description there's nothing I can google to figure it out

Reply:
Definitely.

Reply:
"Deviated the most"?

Reply:
"Sort graphs by deviation" would be better, also since there is a sort order I'm not sure we want to say "the most" because when sorting DESC you're seeing "the least"

Reply:
Yeah, great point there. I removed the "first" from other descriptions, because first and last will be defined by the ascending or descending order. Thanks!